src/classes/TestingUtils.cls

```diff
@@ -0,0 +1,40 @@
+/*************************************************************************************************
+* This file is part of the Nebula Framework project, released under the MIT License.             *
+* See LICENSE file or go to https://github.com/jongpie/NebulaFramework for full license details. *
+*************************************************************************************************/
+public class TestingUtils {
```

Comment:
This class is pretty baller. Since it's meant to be used only in tests, what do you think about adding something like this to it?
```
public class TestingUtils {
    static {
        if(!Testing.isRunningTest()) throw new Exceptions.InvalidOperationException('Nah, no way, sorry');
    }
...
```
This would prevent someone from using anything in the TestingUtils class - it would only run in a test context. What do you think - overkill?